lib/gstreamer/va/transcoder.py

```diff
@@ -41,6 +41,9 @@ def make_requirements():
       Codec.VP9 : dict(
         hw = (platform.get_caps("decode", "vp9_8"), have_gst_element(f"va{hw}vp9dec"), f"matroskademux ! vp9parse ! va{hw}vp9dec"),
       ),
+      Codec.AV1 : dict(
+        hw = (platform.get_caps("decode", "av1_8"), have_gst_element(f"va{hw}av1dec"), f"matroskademux ! av1parse ! va{hw}av1dec"),
+      ),
```

Comment:
This is already defined/implemented for gst-va (see right after the HEVC definition).